docs/application/dotnet/guides/nui/view.md

```diff
@@ -1,75 +1,133 @@
+<a name="2"></a>
+## Directional Navigation
+
+NUI also provides directional navigation between `View` using arrow keys on the keyboard. You can simply set the following properties to specify the next `FocusableViews` for each direction:
+
+- [View.UpFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_UpFocusableView)
+
+- [View.DownFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_DownFocusableView)
+
+- [View.LeftFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_LeftFocusableView)
+
+- [View.RightFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_RightFocusableView)
+
+When an arrow key is pressed and the current `View` has next `FocusableView` for the direction, the system passes focus to the next `FocusableView`. However, if the current `View` does not have the next `FocusableView` for the direction or the next `ForcusableView` is not on the window, the property returns `NULL` and no change occurs.
 
-![TextLabel object with a red background](./media/background_textlabel.png)
 
 <a name="3"></a>
-## Setting the Background Image
+## View Event
 
-You can set a background image of a view:
+`View` provides a variety of `events` that are commonly used by the components. The following table lists the basic events provided by `View`:
 
-```
-View view = new View();
-view.BackgroundImage = "image.png";
+**Table: View events**
+
+| Event               | Description                                               |
+| ------------------- | --------------------------------------------------------- |
+| `KeyEvent`          | Emitted when the key event is received.                   |
+| `TouchEvent`        | Emitted when the touch input is received.                 |
+| `HoverEvent`        | Emitted when the hover input is received.                 |
+| `WheelEvent`        | Emitted when the wheel event is received.                 |
+| `FocusGained`       | Emitted when the control gets the key input focus.        |
+| `FocusLost`         | Emitted when the control loses key input focus.           |
+| `Relayout`          | Emitted after the size has been set on View during relayout. |
+| `LayoutDirectionChanged` | Emitted when the layout direction property of View or its parent View is changed. |
+| `AddedToWindow`     | Emitted after View has been connected to the window.  |
+| `RemovedFromWindow` | Emitted after View has been disconnected from the window. |
+| `VisibilityChanged` | Emitted when the visible property of this or a parent View is changed. |
+| `ResourcesLoaded`   | Emitted after all the resources required by View are loaded and ready. |
+
+You can create custom `callback` methods with event Handler for each `event`.
```

Comment:
event Handler 

Is there any reason for the capital letter H in Handler?

Reply:
I think it is a typing mistake.
@mijin85cho Thank you for pointing out.
@bshsqa,
Please change "event Handler" to "event handler".

Reply:
Oh.. I missed it. Thank you.